lib/commands/context.js

```diff
@@ -37,7 +37,7 @@ commands.getContexts = async function () {
 };
 
 commands.setContext = async function (name) {
-  if (name === null) {
+  if (name === null || name === undefined) {
```

Comment:
what about `#hasValue` by `import { util } from 'appium-support';`?
https://github.com/appium/appium-support/blob/2fcf470576f0357e3200f8014f8b1898b326d79e/lib/util.js#L14

Reply:
ooh that's much better. I love that.